randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -1484,7 +1484,9 @@ Extra - asset_id: collision_camera_021
           Use Spin Boost
           Diagonal Bomb Jump (Intermediate) and Lay Normal Bomb
           Flash Shift and Speed Booster and After Ghavoran - Ammo Recharge Enky Above and Speedbooster Conservation (Intermediate)
-          Flash Shift and Movement (Advanced)
+          All of the following:
+              # https://www.youtube.com/watch?v=rQiQbXvvCSA
+              Flash Shift and Movement (Advanced)
```

Comment:
I would change Movement to Climb Sloped Tunnels, and possibly throw in a Walljump Intermediate or Advanced with it.

Reply:
Done.